src/api/spec/support/shared_examples/features/beta/user_tab.rb

```diff
@@ -14,11 +14,19 @@ def toggle_checkbox(html_id)
     # Thus we have to remove bootstraps custom-control classes.
     page.execute_script("$('##{html_id}').removeClass('custom-control-input')")
     page.execute_script("$('label[for=#{html_id}]').removeClass('custom-control-label')")
+    scroll_page(html_id) if mobile?
     find_field(html_id).click
     # FIXME: Needed to wait for the Ajax call to perform
     sleep(1)
   end
 
+  def scroll_page(html_id)
```

Comment:
We don't run this on mobile anymore right?

Reply:
In line 17, yes.

Reply:
You use the method `toggle_checkbox` only in the tests you skip right?

Reply:
And if not, why don't you skip those tests on mobile too?

Reply:
> You use the method `toggle_checkbox` only in the tests you skip right?

`toggle_checkbox` is not only used in the tests that are skipped.

> And if not, why don't you skip those tests on mobile too?

Because I prefer to test as much feature tests as possible on mobile. Why should we skip those tests on mobile?

Reply:
You're lucky, man. We merged Bootstrap 4.5.0 today. It changed how collapsible datatable rows are shown. Now they are correctly collapsed:

![Screenshot from 2020-05-20 12-48-19](https://user-images.githubusercontent.com/24919/82437774-431e7f00-9a98-11ea-86fd-da4ceefd1be2.png)

There is no need to define a `scroll_page` method, for now. :wink: